.github/shared/src/swagger.js

```diff
@@ -64,6 +64,36 @@ export class Swagger {
     return this.#refs;
   }
 
+  /**
+   * @returns {Promise<Set<Swagger>>}
+   */
+  async getExamples() {
+    if (!this.#refs) {
```

Comment:
Refactor to store all the refs (examples and swaggers) in `this.#refs`, then have the get methods return a filtered view of the map.

Reply:
Definite woopsy. This is done now @mikeharder